2009-flu-uk.qmd

```diff
@@ -1 +1,437 @@
+original_data <- read.csv(file = "data/andre_estimates_21_02.txt", sep = "\t") %>% rowSums()
+
+# Group all age groups
+data <- data.frame(Cases = original_data) %>%
+  mutate(Days = seq(7, by = 7, length.out = n())) %>%
+  dplyr::select(Days, Cases)
+```
+
+### Plotting the data
+
+You can plot the data and see the two waves of infections and the presence of holiday periods.
+
+```{r}
+# Set up the plot
+plot(data$Day, data$Cases, pch = 19, col = "red", 
+     xlab = "Days since start of epidemics",
+     ylab = "Official cases count",
+     main = "Cases over Time with Holiday Periods",
+     ylim = c(0, max(data$Cases) * 1.3), # Add padding to the y-axis
+     xlim = c(min(data$Day), max(data$Day)))
+
+# Add the holiday periods
+hol_t <- c(0, 38, 85, 140, 147) # Holiday timings
+hol_v <- c(1, 0.70, 1, 0.85, 1) # Contact reduction
+
+# Summer holidays
+rect(xleft = hol_t[2], ybottom = 0, xright = hol_t[3], 
+     ytop = max(data$Cases) * 1.2,
+     col = "#87CEEB77", border = NA)
+     #col = adjustcolor("skyblue", alpha.f = 0.3), border = NA)
+text(x = mean(hol_t[2:3]), y = max(data$Cases) * 1.25, 
+     labels = "Summer Holiday", col = "blue", cex = 0.8)
+
+# Half-term holiday
+rect(xleft = hol_t[4], ybottom = 0, xright = hol_t[5], 
+     ytop = max(data$Cases) * 1.2, 
+     col = "#90EE9077", border = NA)
+text(x = mean(hol_t[4:5]), y = max(data$Cases) * 1.25, 
+     labels = "Half-Term", col = "darkgreen", cex = 0.8)
+
+# Legend
+legend("topright", legend = c("Cases", "Summer Holiday", "Half-Term"),
+       pch = c(19, NA, NA), col = c("red", "#87CEEB77", "#90EE9077"),
+       lty = c(NA, 1, 1), lwd = c(NA, 10, 10), 
+       bg = "white", inset = 0.02, bty = "n")
+
+```
+
+
+## Model design
+
+### Transmission model
+
+Similarly to the SIR model, we saw already, the SEIR model divides the population into four compartments:
+
+- **S**usceptible: Individuals at risk of infection.  
+- **E**xposed: Infected individuals who are not yet infectious.  
+- **I**nfectious: Actively transmitting the disease.  
+- **R**ecovered: Immune individuals.  
+
+Additionally, the model incorporates time-varying transmission rates to account for changes in contact patterns during holidays.
+
+The SEIR model is governed by the following system of ordinary differential equations:
+
+$$
+\left\{
+\begin{aligned}
+    \frac{dS}{dt} &= - \beta(t) \frac{S I}{N} \\
+    \frac{dE}{dt} &= \beta(t) \frac{S I}{N} - \gamma E \\
+    \frac{dI}{dt} &= \gamma - \sigma I \\
+    \frac{dR}{dt} &= \sigma I
+\end{aligned}
+\right.
+$$
+
+where:
+
+- $\beta(t)$ is the time-dependent transmission rate, representing the rate at which susceptible individuals become exposed through contact with infectious individuals.
+- $\gamma$ is the rate at which exposed individuals transition to the infectious stage (the inverse of the incubation period).
+- $\sigma$ is the recovery rate, indicating the rate at which infectious individuals recover or are removed from the infectious population.
```

Comment:
Is it worth swapping gamma and sigma so that gamma is the recovery rate, to be consistent with earlier sections?

Reply:
agree. Similarly we might change Infected to Infectious in earlier sections to match here

Reply:
Done - re swapping sigma and gamma to be consistent with the rest of the book. Re Infected vs Infectious - I guess the previous example is a SIR where infected = infectious, and when it is more natural to refer to infected for the compartment. In the SEIR we want to distinguish between the "infected" E+I and the "infectious" only I so maybe keep as it is?